kubevirtci

```diff
@@ -185,6 +187,10 @@ function kubevirtci::create_cluster() {
 
 	echo "Waiting for worker VM in tenant cluster namespace"
 	kubevirtci::retry_until_success kubevirtci::vm_matches "${TENANT_CLUSTER_NAME}-md-"
+
+	if [ ${remove_template} ]; then
+	  rm -rf templates
```

Comment:
two things.
1. formatting looks off.
2. i don't think we should be messing with removing this directory unless we use `mktemp`

Reply:
Removed.